hathor/indexes/manager.py

```diff
@@ -63,6 +64,22 @@ def enable_tokens_index(self) -> None:
         """Enable tokens index. It does nothing if it has already been enabled."""
         raise NotImplementedError
 
+    def _manually_initialize(self, tx_storage: 'TransactionStorage') -> None:
+        """ Initialize the tips indexes, populating them from a tx_storage that is otherwise complete.
```

Comment:
Maybe add a note that this method is still not being used?

Reply:
Done.